src/index.html

```diff
@@ -4,9 +4,369 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img class="header__logo" src="./images/logo.png"
+        alt="moyo">
```

Comment:
You need to check and fix it everywhere
<img width="1008" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/47dfb70e-fe0b-47b8-be64-fd69db28890f">

```suggestion
        <img
          class="header__logo"
          src="./images/logo.png"
          alt="moyo"
        >
```